modules/ROOT/pages/kubernetes/persistent-volumes.adoc

```diff
@@ -158,6 +162,8 @@ When this method is used, the `persistentVolumeClaim` object must already exist.
 ----
 volumes:
   backups:
+  labels:
+        backup: true
     mode: volume
```

Comment:
here the mode is type volume so the label will not work

Reply:
we can remove the labels from here

Reply:
aaah, good catch!